frontend/internal-packages/agent/src/db-agent/nodes/invokeSchemaDesignToolNode.ts

```diff
@@ -24,7 +24,10 @@ export const invokeSchemaDesignToolNode = async (
   const result = await toolNode.invoke(state, {
     configurable: {
       ...config.configurable,
+      buildingSchemaId: state.buildingSchemaId,
+      latestVersionNumber: state.latestVersionNumber,
       buildingSchemaVersionId: state.buildingSchemaVersionId,
+      designSessionId: state.designSessionId,
```

Comment:
I believe that it would be more efficient to pass the database object as is, rather than passing multiple IDs. This will be addressed separately.